### PR TITLE
Setup lerna workflow

### PR DESCRIPTION
**Pull request by koonopek**
After relasing, we should change in all other repos to use this package

### CODE REVIEW

File: packages/cache-service/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redstone-finance/cache-service",
-  "version": "0.0.1",
+  "version": "1.0.1",
```
Comment (hatskier) [inline packages/cache-service/package.json:3]: Why do you want to bump the major version? Can't we just increase the minor one to have e.g. 0.3.0 everywhere?

**Review by hatskier — COMMENTED** [review]
Good job! Could you please also add few words about using lerna somewhere (e.g. in README or to our wiki)?

File: packages/cache-service/package.json (hunk 1 above)
Comment (koonopek) [reply packages/cache-service/package.json:3]: we can, I don't why.

How is 0.3.0 better then 1.0.1?

**Review by Lukasz2891 — COMMENTED** [review]
That's ok for me regarding to the code - more general questions on dscrd

File: package.json [hunk 2]
```diff
@@ -1,40 +1,29 @@
...
-      "packages/oracle-node",
-      "packages/oracles-smartweave-contracts",
-      "packages/protocol",
-      "packages/rpc-providers",
-      "packages/sdk",
-      "packages/starknet-connector",
-      "packages/starknet-relayer",
-      "packages/streamr-proxy",
-      "packages/utils"
+      "packages/*"
```
[the remark this answers is not in the document]
Comment (koonopek) [reply package.json:26]: yep!

File: packages/cache-service/test/e2e/data-packages.spec.ts [hunk 3]
```diff
@@ -8,11 +8,11 @@ import {
   SignedDataPackagePlainObj,
   consts,
   utils,
-} from "redstone-protocol";
+} from "@redstone-finance/protocol";
 import {
   RedstonePayloadParser,
   convertDataPointToNumericDataPoint,
-} from "redstone-protocol/dist/src/redstone-payload/RedstonePayloadParser";
+} from "@redstone-finance/protocol/dist/src/redstone-payload/RedstonePayloadParser";
```
[the remark this answers is not in the document]
Comment (koonopek) [reply packages/cache-service/test/e2e/data-packages.spec.ts:15]: Yeah, i think it should be fixed but for sure not in this PR

**Review by cehali — APPROVED** [review]
Good job, probably we should wait for eslint PR to merged first

File: packages/cache-service/package.json (hunk 1 above)
Comment (koonopek) [reply packages/cache-service/package.json]: Okey it is opinionated but set to 0.3.0 to merge

**Review by hatskier — APPROVED** [review]

**Pull request merged**